api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -69,29 +69,89 @@ public void sendBetaAccessRequestEmail(String userName) throws MessagingExceptio
   }
 
   @Override
-  public void sendWelcomeEmail(String contactEmail, String password, User user)
+  public void sendWelcomeEmail(final String contactEmail, final String password, final User user)
       throws MessagingException {
-    try {
-      InternetAddress email = new InternetAddress(contactEmail);
-      email.validate();
-    } catch (AddressException e) {
-      throw new MessagingException("Email: " + contactEmail + " is invalid.");
-    }
-    MandrillMessage msg = buildWelcomeMessage(contactEmail, password, user);
+    final String msgHtml =
+        buildHtml(WELCOME_RESOURCE, welcomeMessageSubstitutionMap(password, user));
+    final MandrillMessage msg = buildMessage(contactEmail, msgHtml, "Your new All of Us Account");
     sendWithRetries(msg, String.format("Welcome for %s", user.getName()));
   }
 
   @Override
-  public void sendBetaAccessCompleteEmail(String contactEmail, String username)
+  public void sendBetaAccessCompleteEmail(final String contactEmail, final String username)
       throws MessagingException {
+    final String msgHtml = buildHtml(BETA_ACCESS_RESOURCE, betaAccessSubstitutionMap(username));
+    final MandrillMessage msg =
+        buildMessage(contactEmail, msgHtml, "All of Us ID Verification Complete");
```

Comment:
mega nit: I feel like subject, body parameter order is more intuitive than body, subject